### ISSUE TITLE
Do not ignore the return vale of snapraid diff

### ISSUE THREAD

**Issue by Chronial** [Chronial/snapraid-runner#12]
At the moment we ignore the return value of `snapraid diff`, because it used to be useless. This is not the case anymore, so we should check for errors.

**Comment by deafboy** [comment]
I second this.

Doesn't look like a success:

```
SnapRAID job completed successfully:


2018-12-28 07:10:02,315 [INFO  ] ============================================================
2018-12-28 07:10:02,324 [INFO  ] Run started
2018-12-28 07:10:02,324 [INFO  ] ============================================================
2018-12-28 07:10:02,325 [INFO  ] Running diff...
2018-12-28 07:10:02,716 [OUTERR] Error reading directory '/snapraid/snap_data2/'. Input/output error.
2018-12-28 07:10:02,716 [OUTERR] You can exclude it in the config file with:
2018-12-28 07:10:02,717 [OUTERR] exclude /
2018-12-28 07:10:03,021 [INFO  ] ************************************************************
2018-12-28 07:10:03,022 [INFO  ] Diff results: 0 added,  0 removed,  0 moved,  0 modified
2018-12-28 07:10:03,023 [INFO  ] No changes detected, no sync required
2018-12-28 07:10:03,023 [INFO  ] All done
```

**Comment by Chronial** [comment]
Seems like I fixed this a while ago in 6262edd11b4f495ca45323a101801c7209ce5adc.
